eng/tools/RepoTasks/RepoTasks.csproj

```diff
@@ -23,6 +23,8 @@
     <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkVersion)" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildTasksCoreVersion)" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCoreVersion)" />
+    <!-- Manually updated version from 6.0.0 to address CVE-2021-43877 -->
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="6.0.1" />
```

Comment:
We should property-ize this and put it in `Versions.props`

Reply:
Done. It differs from the 8.0 version so I chose a different property name